modules/ROOT/pages/afm-delete-schedule.adoc

```diff
@@ -7,33 +7,33 @@ endif::[]
 
 You might want to delete a schedule for any of the following reasons:
 
-* You are finished testing endpoints of a public API and you want to free resources for testing endpoints of a different public API.
-* You are finished testing endpoints of a private API.
+* You have finished testing endpoints of a public API and you want to free resources for testing endpoints of a different public API.
+* You have finished testing endpoints of a private API.
 * You have more than one schedule running the same test and you no longer need one of the schedules.
 
-[IMPORTANT]
+[NOTE]
 ====
-Be sure that you want to delete a schedule. AFM does not give you a chance to confirm that you want to delete it.
+Be sure that you want to delete the schedule. There is no opportunity to confirm the deletion.
 ====
 
 == Procedure
 
-There are two different methods for deleting a schedule.
+Anypoint Functional Monitoring provides two methods for deleting a schedule.
 
-To delete a schedule from the *Schedules* section of a monitor:
+To delete a schedule from the *Schedules* table of a monitor:
 
-. Select the monitor in which you want to modify a schedule.
-. Click the *Schedules* tab.
-. Find the schedule in the table, click the three dots on the right end of the table row, and select *Delete*.
+. Select the monitor that contains a schedule to delete.
+. Locate the schedule in the table. Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Delete Schedule*.
```

Comment:
The main action here is the second sentence, which bothers me a little.
There are three actions here: locate, click, and click.
This can be worded in one, two, or three steps.
I think the best is:
```suggestion
. Locate the schedule in the table and click *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) > *Delete Schedule*.
```